package.json

```diff
@@ -43,6 +43,14 @@
     "hubot": "./bin/hubot"
   },
   "scripts": {
-    "test": "./script/test"
+    "clean": "rm -rf lib/",
+    "preinstall": "script/dotenv",
```

Comment:
I'm not sure exactly.

I think at one point, you could just run hubot out of a clone of this repository. That introduced some problems where people would fork this (the framework) repository, and put their scripts in it, rather than using it as a framework. I think that was undone at one point, but maybe `.env` here is an artifact of that?